.github/workflows/test.yml

```diff
@@ -24,7 +24,7 @@ jobs:
 
       - run: npm install
 
-      - run: npm test
+      - run: npm run test-coverage
```

Comment:
Maybe better to just run `npm test` instead of coverage, and run coverage as a separate job? That also makes it easier to report on coverage

Reply:
i just translated what was still in the `.travis.yml` and converted the `test-travis` script to `test-coverage`

personally i would (like in mongoose) have coverage and normal testing in one step, because that net-reduces CI time and steps to maintain (and chance of failure because it does not have to run another whole test suite)